apps/molgenis-components/src/components/forms/InputRangeInt.vue

```diff
@@ -1,29 +1,40 @@
 <template>
   <FormGroup :id="id + '-from'" :label="label" :description="description">
-    <InputInt
-      :id="id + '-from'"
-      :value="value[0]"
-      @input="emitValue($event, 0)"
-      placeholder="from"
-      class="m-0 d-inline-block"
-    />
-    <InputInt
-      :id="id + '-to'"
-      :value="value[1]"
-      @input="emitValue($event, 1)"
-      placeholder="to"
-      class="m-0 d-inline-block"
-    />
+    <InputGroup class="d-flex">
+      <template v-slot:prepend>
+        <slot name="prepend"></slot>
+      </template>
+    
+        <BaseIntInput
+          :id="id + '-from'"
+          :value="value[0]"
+          @input="emitValue($event, 0)"
+          placeholder="from"
+          class="m-0"
```

Comment:
Without the "m-0" in both BaseIntInput components, bootstrap will connect the fields together (removing the double line we talked about)

And i don't think we still need the "m-0" here anymore anyway.

Reply:
will test removing the m-0